packages/scorer/src/provider/model-graded/faithfulness.test.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, test } from "vitest";
+import { checkLlmFaithfulness } from "./faithfulness";
+
+test("llm-faithfulness works", async () => {
```

Comment:
**Add more unit tests to the `llm-faithfulness` scorer to improve coverage and reliability.** 
 The `checkLlmFaithfulness` function is a critical part of the `llm-faithfulness` scorer, and it would be beneficial to add more unit tests to cover different scenarios and edge cases. This will help ensure the scorer's reliability and maintainability. [Testing]
```suggestion
// Add more test cases to cover different scenarios, such as:
// - Empty context or claims
// - Partially correct claims
// - Completely incorrect claims
// - Handling of edge cases and errors
test("llm-faithfulness handles empty context", async () => {
  const [scoreResult] = await checkLlmFaithfulness({
    sample: {
      id: "1",
      inputs: {},
    },
    output: {
      value: "",
    },
    config: {
      type: "llm-faithfulness",
      context: "",
      claims: ["John is a student at XYZ University."],
    },
  });
  expect(scoreResult?.score).toBe(0);
  expect(scoreResult?.message).toContain("Failed to call LLM");
  expect(scoreResult?.name).toBe("llm-faithfulness");
});
```